### ISSUE TITLE
Missing source and javadoc artifacts on Maven Central for xmpcore 6.1.11

### ISSUE THREAD

**Issue by don-vip** [adobe/XMP-Toolkit-SDK#41]
### Expected Behaviour
- Each version of xmpcore published on Maven Central should contain source and javadoc artifacts

### Actual Behaviour
- OK for version 6.1.10: https://repo1.maven.org/maven2/com/adobe/xmp/xmpcore/6.1.10/
- KO for version 6.1.11: https://repo1.maven.org/maven2/com/adobe/xmp/xmpcore/6.1.11/

**Comment by Lonzak** [comment]
@don-vip See https://github.com/adobe/XMP-Toolkit-SDK/issues/5

**Comment by lrosenthol** [comment]
The 6.1.11 version of the XMP Toolkit at https://search.maven.org/artifact/com.adobe.xmp/xmpcore/6.1.11/bundle is an official release and should be fine.

**Comment by Lonzak** [comment]
Yes the version itself may be fine (aka usable) however in contrast to version [6.1.10](https://repo1.maven.org/maven2/com/adobe/xmp/xmpcore/6.1.10/) the [sources are missing](https://repo1.maven.org/maven2/com/adobe/xmp/xmpcore/6.1.11/).
